03-task-manager/starter/app.js

```diff
@@ -1 +1,35 @@
-console.log('Task Manager App')
+const express = require('express');
+const app = express();
+const tesks = require('./routes/tasks');
```

Comment:
nit: typo tasks

Reply:
I got it. tesks is going to be tasks :).
I solve it. Thanks a lot.